modules/nw-dns-forward.adoc

```diff
@@ -55,6 +63,9 @@ spec:
 ----
 <1> Must comply with the `rfc6335` service name syntax.
 <2> Must conform to the definition of a subdomain in the `rfc1123` service name syntax. The cluster domain, `cluster.local`, is an invalid subdomain for the `zones` field.
+ifdef::openshift-rosa,openshift-dedicated[]
+Only forward to specific zones, such as your intranet. You must specify at least one zone. Otherwise, your cluster can lose functionality.
```

Comment:
Question: is there a reason that `You must specify at least one zone. Otherwise, your cluster can lose functionality.` is not encapsulated in an IMPORTANT admonition like you have in line 16 above? Does it appear too messy when a note is indented under a callout label?